libraries/AP_IOMCU/iofirmware/iofirmware.cpp

```diff
@@ -447,7 +567,7 @@ void AP_IOMCU_FW::update()
     loop_counter++;
 
     if (do_reboot && (last_ms > reboot_time)) {
-        hal.scheduler->reboot(true);
+        hal.scheduler->reboot(false);
```

Comment:
why?

Reply:
ok sounds reasonable. Probably needs testing on other boards that have iofirmware update issues - particularly Pixhawk6X